docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -122,51 +122,168 @@ class Sidebar extends React.Component<any, any> {
     })
 */
 
+  getActiveCategoryIndex(at: String, sections: ShorthandValue<any>[]) {
+    let i
+    for (i = 0; i < sections.length; i++) {
+      const category = sections[i]
+      if (!('items' in category)) {
+        continue
+      }
+      if (category.items.filter(c => 'title' in c).some(e => e.title.to === at)) {
+        return i
+      }
+    }
+    return -1
+  }
+
+  keyUpCallback(e) {
+    if (e.key !== 'Enter') {
+      return
+    }
+    e.target.click()
```

Comment:
this should work out of the box in Tree, can you please create an issue?

Reply:
Issue #1613